react/src/components/ResourceAllocationFormItems.tsx

```diff
@@ -951,7 +738,10 @@ const ResourceAllocationFormItems: React.FC<
                             },
                           }}
                           min={0}
```

Comment:
I think we need to apply disabled prop according to `currentImageAcceleratorLimits.length <= 0` since AI accelerator is dependent to image. (please refer [here](https://github.com/lablup/backend.ai-webui/pull/2108/files#diff-19c600075bfc029d497f5d6514affbea172d6e5d9c2a194a05a755241e72636fL111-L115))